test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```

Comment:
What is the test helper?

Reply:
I believe it's automatically generated when you create a controller